internal/cluster/validator.go

```diff
@@ -566,3 +573,36 @@ func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) (V
 	return ValidationFailure, fmt.Sprintf("Hosts' clocks are not synchronized (there's more than a %d minutes gap between clocks), "+
 		"please configure an NTP server via DHCP or set clocks manually.", common.MaximumAllowedTimeDiffMinutes)
 }
+
+// Ignore hostPrefix set to 0 for non-OVN/SDN plugins.
+func (v *clusterValidator) skipNetworkHostPrefixCheck(c *clusterPreprocessContext, prefix int64) bool {
+	// list of known plugins that require hostPrefix to be set
+	var pluginsUsingHostPrefix = []string{models.ClusterNetworkTypeOVNKubernetes, models.ClusterNetworkTypeOpenShiftSDN}
+
+	if prefix != 0 {
```

Comment:
what this means means? Didn't understand to say the truth

Reply:
The thought was to skip the validation *only* if the prefix was not set and the networkTypes was not OVN of SDN.
If we take the approach that the validation should be skipped regardless of the prefix then this check can be removed (and as above the check can be moved).

Reply:
Updated to skip validations only based on networkType and ignore the prefix.